File: codekata/mathematics/power_of_2.py
# given a number N, check whether it is a power of 2
# get input N from user
N=int(input())
# define power function
def power():
# if condition checks the power of 2
 if(N%2==0):
   c="yes"
   return c
 else:
   c="no"
   return c
# call the power function and store it in result
result=power()
# print the result value
print(result)

File: codekata/javascript/mathematics/power_of_2.js
// Given a number N, check whether it is a power of 2
const readline=require('readline');
const inp=readline.createInterface({
    input:process.stdin
});
const userInput=[];
inp.on("line",(data)=>{
    userInput.push(data);
});
inp.on("close",()=>{
    var N=userInput[0];
 if(N%2===0)
   console.log("yes");
 else
   console.log("no");

});
